flexmeasures/data/migrations/versions/650b085c0ad3_consolidate_data_source_after_storage_.py

```diff
@@ -0,0 +1,27 @@
+"""Consolidate data source after storage schedulers merged
+
+Revision ID: 650b085c0ad3
+Revises: 30f7b63069e1
+Create Date: 2022-11-16 07:07:44.281943
+
+"""
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = "650b085c0ad3"
+down_revision = "30f7b63069e1"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.execute(
+        "Update data_source set version='1', model='StorageScheduler' where name = 'Seita' and type='scheduling script';"
```

Comment:
Does it matter whether `Update` is capitalized?

Reply:
No